tests/backup/backup_test.go

```diff
@@ -34,6 +34,18 @@ var _ = Describe("{BackupClusterVerification}", func() {
 	})
 })
 
+var _ = Describe("{testParseConfig}", func() {
```

Comment:
Can we remove this before merging?

Reply:
discussed with mithun will add a comment to keep it as reference test